WooCommerce/Classes/ViewRelated/Products/Cells/ProductsTabProductTableViewCell.swift

```diff
@@ -64,7 +64,14 @@ extension ProductsTabProductTableViewCell: SearchResultCell {
 
 extension ProductsTabProductTableViewCell {
     func update(viewModel: ProductsTabProductViewModel, imageService: ImageService) {
-        nameLabel.text = viewModel.name
+        if let variationID = viewModel.productVariation?.productVariationID {
+            // Add product variation ID with name
+            // See more: https://github.com/woocommerce/woocommerce-ios/issues/4846
+            nameLabel.text = "\(Localization.variationID(variationID: "\(variationID)"))\n\(viewModel.name)"
+        }
+        else {
+            nameLabel.text = viewModel.name
+        }
```

Comment:
What do you think if instead of having this logic in the view/cell, we move this to the view model, Ideally the view should not be doing calculations and should only be accessing `viewModel.name`. Also, that allows us to unit test that logic easily. Thoughts?

Reply:
You are right, that would be better.